assets/js/modules/tagmanager/setup.js

```diff
@@ -486,37 +532,48 @@ class TagmanagerSetup extends Component {
 						onEnhancedChange={ this.handleAccountChange }
 						outlined
 					>
-						{ accounts.map( ( account ) => {
-							return (
-								<Option
-									key={ account.accountId /* Capitalization rule exception: `accountId` is a property of an API returned value. */ }
-									value={ account.accountId /* Capitalization rule exception: `accountId` is a property of an API returned value. */ }>
-									{ account.name }
-								</Option>
-							);
-						} ) }
+						{ []
+							.concat( accounts )
+							.concat( ! hasExistingTag ? {
+								name: __( 'Set up a new account', 'google-site-kit' ),
+								accountId: ACCOUNT_CREATE, /* Capitalization rule exception: `accountId` is a property of an API returned value. */
+							} : [] )
+							.map( ( account ) => {
+								return (
+									<Option
+										key={ account.accountId /* Capitalization rule exception: `accountId` is a property of an API returned value. */ }
+										value={ account.accountId /* Capitalization rule exception: `accountId` is a property of an API returned value. */ }>
+										{ account.name }
+									</Option>
+								);
+							} )
+						}
 					</Select>
 
 					{ containersLoading ? ( <ProgressBar small /> ) : (
 						<Select
+							className="googlesitekit-tagmanager__select-container"
 							enhanced
 							name="containers"
 							label={ __( 'Container', 'google-site-kit' ) }
 							value={ selectedContainer }
-							disabled={ hasExistingTag }
+							disabled={ hasExistingTag || [ ACCOUNT_CREATE, UNSELECTED ].includes( selectedAccount ) }
 							onEnhancedChange={ this.handleContainerChange }
 							outlined
 						>
-							{ containers.concat( {
-								name: __( 'Set up a new container', 'google-site-kit' ),
-								publicId: 0,
-							} ).map( ( { name, publicId }, i ) =>
-								<Option
-									key={ i }
-									value={ publicId /* Capitalization rule exception: `publicId` is a property of an API returned value. */ }>
-									{ name }
-								</Option>
-							) }
+							{ []
+								.concat( containers )
+								.concat( {
```

Comment:
We should probably add this option only if `! hasExistingTag`, like in the accounts dropdown.

Reply:
Done! 👍